### PR TITLE
apiv2(ticdc): implmenet the GET changefeed

### PR DESCRIPTION
**Pull request by charleszheng44**
### What problem does this PR solve?

Issue Number: ref #8019 

### What is changed and how it works?

Implement the get changefeed for API v2

#### Tests 

 - Unit test
 - Integration test

##### Will it cause performance regression or break compatibility?
No

##### Do you need to update user documentation, design documentation or monitoring documentation?
No

### Release note 
```release-note
None
```

### CODE REVIEW

**Comment by charleszheng44** [comment]
@sdojjy

File: cdc/api/v2/changefeed.go [hunk 1]
```diff
@@ -297,6 +298,82 @@ func (h *OpenAPIV2) updateChangefeed(c *gin.Context) {
...
+// getChangefeed get detailed info of a changefeed
+// @Summary Get changefeed
+// @Description get detail information of a changefeed
+// @Tags changefeed
+// @Accept json
+// @Produce json
+// @Param changefeed_id  path  string  true  "changefeed_id"
+// @Success 200 {object} model.ChangefeedDetail
+// @Failure 500,400 {object} model.HTTPError
+// @Router /api/v1/changefeeds/{changefeed_id} [get]
```
Comment (sdojjy) [inline cdc/api/v2/changefeed.go:310]:
```suggestion
// @Router /api/v2/changefeeds/{changefeed_id} [get]
```

**Comment by sdojjy** [comment]
please fix the ut test cases

**Review by asddongmen — APPROVED** [review]

File: cdc/api/v2/changefeed.go (hunk 1 above)
Comment (charleszheng44) [reply cdc/api/v2/changefeed.go:310]: fixed

**Review by sdojjy — APPROVED** [review]

**Pull request merged**